<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <!--small_office-modify original->pal_office -->
    <arg name="world"  default="protap_warehouse"/>  <!-- pal office, ... (see ../worlds) -->
    <arg name="gui" default="true"/>
    <arg name="debug" default="false"/>
    <arg name="public_sim" default="true"/>
    <arg name="map" default="$(find mult_protap_sim)/maps"/>

    <env name="GAZEBO_MODEL_PATH" value="$(find tiago_gazebo)/models:$(optenv GAZEBO_MODEL_PATH)"/>

    <env name="LC_NUMERIC" value="C" />

    <!-- Gazebo start -->
    <include file="$(find gazebo_ros)/launch/empty_world.launch">
        <arg name="world_name" value="$(find tiago_gazebo)/worlds/$(arg world).world"/>
        <arg name="gui" value="$(arg gui)"/>
        <arg name="debug" value="$(arg debug)"/>
        <arg name="verbose" value="true" />
    </include>

    <!-- Add first tiago -->


    <!-- Add second tiago -->

    <include file="$(find tiago_multi)/launch/launch_tiago_navigation.launch">
        <arg name="robot_name" value="tiago1"/>
        <arg name="x_pose" value="1.4"/>
        <arg name="y_pose" value="-0.7"/>
        <arg name="public_sim" value="$(arg public_sim)"/>
	<arg name="map" value="$(arg map)"/>
    </include>

    <include file="$(find tiago_multi)/launch/launch_tiago_navigation.launch">
        <arg name="robot_name" value="tiago2"/>
        <arg name="x_pose" value="1.4"/>
        <arg name="y_pose" value="0.7"/>
        <arg name="public_sim" value="$(arg public_sim)"/>
	<arg name="map" value="$(arg map)"/>
    </include>

     <!-- <include file="$(find tiago_multi)/launch/launch_tiago_navigation.launch">
        <arg name="robot_name" value="tiago3"/>
        <arg name="x_pose" value="0.4"/>
        <arg name="y_pose" value="0.7"/>
        <arg name="public_sim" value="$(arg public_sim)"/>
	<arg name="map" value="$(arg map)"/>
    </include> -->

    <!--
    <include file="$(find tiago_multi)/launch/launch_tiago_navigation.launch">
        <arg name="robot_name" value="tiago4"/>
        <arg name="x_pose" value="0.4"/>
        <arg name="y_pose" value="-0.7"/>
        <arg name="public_sim" value="$(arg public_sim)"/>
	<arg name="map" value="$(arg map)"/>
    </include>
    -->

    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find mult_protap_sim)/config/rviz/multi_tiago.rviz"/>

</launch>
